### DISCUSSION TITLE
Ubuntu 下怎么打包windows包啊?

### DISCUSSION THREAD

**Discussion by joisun** [tw93/Pake#337] [Q&A] [answered]
我打包后的文件都是linux的, 没有其他平台:
![image](https://user-images.githubusercontent.com/44388103/223609401-7ee42103-e182-4d3a-8caa-dff31c8e8343.png)

**Comment by tw93** [comment, marked as answer]
https://github.com/tw93/Pake/wiki/GitHub-Actions-%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%E5%A4%9A%E7%B3%BB%E7%BB%9F%E7%89%88%E6%9C%AC
用这种方式

**Reply by joisun** [reply]
ok
